bin/run-example.cmd

```diff
@@ -17,6 +17,13 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 rem
 
-set SPARK_HOME=%~dp0..
+rem Figure out where the Spark framework is installed
+set FIND_SPARK_HOME_SCRIPT=%~dp0find_spark_home.py
+if exist "%FIND_SPARK_HOME_SCRIPT%" (
+  for /f %%i in ('python %FIND_SPARK_HOME_SCRIPT%') do set SPARK_HOME=%%i
```

Comment:
I'm not sure if python would be the right one (python2, python3, is it in PATH)?
and I don't think `cmd /c foo.py` works either

Reply:
The assumption was that if `find_spark_home.py` can be found in the local folder, this will be a good setup. While I think `find_spark_home.py` with any Python version (well, at least modern 2 and 3), indeed this not takes into account case that the python is just not there at all.

Reply:
I guess python is generally expected on *nix but quite possibly absent on Windows.